Appointments.Web.Tests/Extensions/DateTimeExtensionsTest.cs

```diff
@@ -11,7 +11,7 @@ public class DateTimeExtensionsTest
         public void ToCustomDateString_returns_a_given_date_in_a_custom_format()
         {
             var date = new DateTime(2015, 07, 15, 14, 14, 00);
-            var result = date.ToCustomDateString();
+            var result = DateTimeExtensions.ToCustomDateString(date);
```

Comment:
You should use revert this change. The proper way to consume this method will be:

```csharp
date.ToCustomDateString();
```